terraform/modules/azure-nat-gateway/outputs.tf

```diff
@@ -1,3 +1,3 @@
 output "public_ip" {
```

Comment:
Maybe add "list"? Would be a breaking change so maybe not wanted.
```suggestion
output "public_ip_list" {
```

Reply:
yup, it is a breaking change BUT this output is not used anywhere (yet?) so LGTM to add it